error.php

```diff
@@ -122,9 +122,9 @@
 // Some nice URLs for getting something for download
 if (preg_match("!^get/([^/]+)$!", $URI, $what)) {
     switch ($what[1]) {
-        case "php"           : $URI = "downloads"; break;
-        case "docs"          : // intentional
-        case "documentation" : $URI = "download-docs"; break;
+        case "php": $URI = "downloads"; break;
+        case "docs": // intentional
+        case "documentation": $URI = "download-docs"; break;
```

Comment:
I'd reformat manually right away:
```suggestion
        case "php":
            $URI = "downloads";
            break;
        case "docs":
        case "documentation":
            $URI = "download-docs";
            break;
```